Graphics/Slicer/Math/Skeleton/NodeTrees.hs

```diff
@@ -77,15 +77,17 @@ pathTo nodeTree@(NodeTree eNodeList@(ENodeList firstENode _) iNodeSets) directio
         iNodeOnThisLevel = findINodeByOutput myINodeSets pLineToFollow False
         iNodeOnLowerLevel = findINodeByOutput (init myINodeSets) pLineToFollow True
         result = findENodeByOutput myENodeList pLineToFollow
-        terminate = ([outOf $ fromJust result], [], fromJust result)
+        terminate = case result of
+                      (Just eNode) -> ([outOf eNode], [], eNode)
+                      Nothing -> error "FIXME: cannot happen."
```

Comment:
Cheating.  :)

But more explicit, so still an improvement.  and if you ever want to get around getting rid of the cheats just don't import `error` anywhere.  Easy, and you can even enforce it with an hlint rule.

Reply:
that's the idea. cheating makes everything more visible. plus, i got to nuke the use of fromJust.